tests/karma.conf.ts

```diff
@@ -0,0 +1,44 @@
+import { BASE_CONFIG } from '../webpack.config';
+import path from 'path';
+
+module.exports = (config: any) => {
+    config.set({
+        autoWatch: false,
+        browserNoActivityTimeout: 10000,
+        concurrency: 1,
+        client: {
+            mocha: {
+                timeout: 8000,
```

Comment:
this should not be needed here, I use it in chemfiles because loading the WASM library takes a while